### DISCUSSION TITLE
Liveries

### DISCUSSION THREAD

**Discussion by vbuff721** [flybywiresim/aircraft#4275] [General]
Are there any liveries available for this new separate add-on package version or do we have to wait for them to be re-made/converted to work with this new package? Or is there some kind of work around to import the liveries that worked with the older versions (i.e. copying the livery folders into the SimObjects>Airplanes folder into the main flybywire aircraft folder)? Thanks for all your hard work on this incredible project!

**Comment by MapleNAV-TCASFAULT** [comment]
In the future, our FBW installer will have a notice that a livery is incompatible and you will have the option to convert them. But for the time being your best bet is to wait it out.

**Reply by vbuff721** [reply]
gotcha! Thanks!

**Reply by CptBayer** [reply]
www.flightsim.to

own category in aircraft for FBW Airbus !!!

Greets Klaus

**Reply by vbuff721** [reply]
Thank you so much for that!

Happy flying!
